corehq/apps/translations/views.py

```diff
@@ -95,7 +95,10 @@ def download_bulk_app_translations(request, domain, app_id):
     lang = request.GET.get('lang')
     skip_blacklisted = request.GET.get('skipbl', 'false') == 'true'
     app = get_app(domain, app_id)
-    headers = get_bulk_app_sheet_headers(app, lang=lang, eligible_for_transifex_only=skip_blacklisted)
+    # if there is a lang selected, assume that user wants a single sheet
+    single_sheet = bool(lang)
```

Comment:
Now that `single_sheet` exists, it'd be more readable to also use it in the `if lang` check below.